examples/unix_socket_server/source/app.d

```diff
@@ -0,0 +1,20 @@
+import vibe.inet.url;
+import vibe.http.server;
+import vibe.http.router;
+import std.stdio;
```

Comment:
Use `vibe.core.log` for logging and in this case this looks like a left over.

Reply:
Yeah sorry, that's a leftover, will fix that.